CONTRIBUTING_GUIDELINES.md

```diff
@@ -0,0 +1,130 @@
+# How to Contribute to Jaeger
+
+We'd love your help!
+
+Jaeger is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+# Certificate of Origin
+
+By contributing to this project you agree to the [Developer Certificate of
+Origin](https://developercertificate.org/) (DCO). This document was created
+by the Linux Kernel community and is a simple statement that you, as a
+contributor, have the legal right to make the contribution. See the [DCO](DCO)
+file for details.
+
+## Making A Change
+
+*Before making any significant changes, please open an
+issue.* Discussing your proposed
+changes ahead of time will make the contribution process smooth for everyone.
+
+Once we've discussed your changes and you've made your updates, then open your PR. Your
+pull request is most likely to be accepted if it:
+
+* Has a [good commit message](https://chris.beams.io/posts/git-commit/):
+    * Separate subject from body with a blank line
+    * Limit the subject line to 50 characters
+    * Capitalize the subject line
+    * Do not end the subject line with a period
+    * Use the imperative mood in the subject line
+    * Wrap the body at 72 characters
+    * Use the body to explain _what_ and _why_ instead of _how_
+* Uses the active voice (vs. [passive voice](https://www.grammarly.com/blog/a-scary-easy-way-to-help-you-find-passive-voice/)) to make it clear when the user has to perform an action and when actions happen automatically.
+* Has been spellchecked and uses proper [grammar](https://www.grammarly.com/).
+* Has been written in plain language and avoids jargon. Remember, the people reading the documentation know much less about the project than you do.
```

Comment:
these 2 lines - same, not a general guideline. We should use CONTRIBUTING from the main repo as the starting point.